### PR TITLE
Landscape support

### PR DESCRIPTION
**Pull request by sleepwalkerfx**
Added the landscape support for SKStoreProductViewController that used to crash on landscape mode. However caption and install button overlaps on some devices in the UI in landscape mode, that needs to be addressed and it's not fixed.

### CODE REVIEW

File: Pod/Classes/TAStoreProductViewController.m [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+#import "TAStoreProductViewController.h"
+
+@interface TAStoreProductViewController ()
+
+@end
+
+@implementation TAStoreProductViewController
+
+- (void)viewDidLoad {
```
Comment (JanC) [inline Pod/Classes/TAStoreProductViewController.m]: please remove the unused methods

File: Pod/Classes/TAStoreProductViewController.m [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    UIInterfaceOrientationMask applicationSupportedOrientations = [[UIApplication sharedApplication] supportedInterfaceOrientationsForWindow:[[UIApplication sharedApplication] keyWindow]];
+    UIInterfaceOrientationMask viewControllerSupportedOrientations = [self supportedInterfaceOrientations];
+    return viewControllerSupportedOrientations & applicationSupportedOrientations;
+}
+
+/*
+ #pragma mark - Navigation
+ 
+ // In a storyboard-based application, you will often want to do a little preparation before navigation
+ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
```
Comment (JanC) [inline Pod/Classes/TAStoreProductViewController.m]: remove comments

**Review by JanC — COMMENTED** [review]
Hi, 
thanks for the pull request. Can you please fix my comments and also remove the `DS_Store` file?

cheers

**Comment by sleepwalkerfx** [comment]
Hi,
I just made the changes.
Thanks.

**Comment by JanC** [comment]
thanks, 
does the `SKStoreProductViewController` crash on some specific iOS versions?

**Comment by sleepwalkerfx** [comment]
Yes it crashed on iOS 10 only when orientation is set to Landscape-only mode. 

>  Terminating app due to uncaught exception 'UIApplicationInvalidInterfaceOrientation', reason: 'Supported orientations has no common orientation with the application, and shouldAutorotate is returning YES'

**Pull request merged**